vms/avm/environment_test.go

```diff
@@ -77,12 +77,6 @@ var (
 
 	keys  = secp256k1.TestKeys()[:3] // TODO: Remove [:3]
 	addrs []ids.ShortID              // addrs[i] corresponds to keys[i]
-
-	noFeesTestConfig = &config.Config{
-		EUpgradeTime:     mockable.MaxTime,
-		TxFee:            0,
-		CreateAssetTxFee: 0,
-	}
```

Comment:
one of the key point of this PR: no more zero fees config. With the E upgrade we won't have zero fees anymore